### DISCUSSION TITLE
Are you planning to add Uyghur to the supported language list?

### DISCUSSION THREAD

**Discussion by neouyghur** [openai/whisper#1889] [General]
Hello, I'm interested in knowing whether there are plans to include Uyghur in the list of supported languages. Additionally, I am curious about the reason behind the absence of a tokenizer for Uyghur. Thank you!

**Comment by Abdurahman-Amat** [comment]
@neouyghur you are totally right! Actually the publicly available open Dataset of the Uyghur language from CommonVoice https://commonvoice.mozilla.org/ug/datasets (dateded: 14.09.2023 and since then added a lot) should be enough to be listed and added to the OpenAI Whisper Tokenizer. This Uyghur dateset is already being used by Speechmatics https://speechmatics.com/ and there is also a demo Video by an Uyghur computer YouTube channel https://www.youtube.com/watch?v=JnxOuaJINwM (Speech to Text is demonstrated at 2:07 of the video). 
Since this demo video was uploaded there is a significant increase in the dataset (see the screenshot at below is from CommonVoice)
**So, please add the Uyghur to the Whisper tokenizer ....**

![e13eaaf0-0edd-4dc2-9309-89a6da1b875f](https://github.com/openai/whisper/assets/49624041/65d51a03-68d6-4487-a509-579551874c7b)
